debian/copyright

```diff
@@ -1,9 +1,19 @@
 Format: http://dep.debian.net/deps/dep5
 Upstream-Name: nest-simulator
-Source: https://github.com/nest/nest-simulator/archive/master.zip
+Source: https://github.com/nest/nest-simulator/
+
+Files: *
+Copyright: 2004-2019 The NEST Initiative <nest_user@nest-initiative.org>
```

Comment:
This is the email address of the old mailing list. And even if it were the new one, it would not be very useful, as sending to it requires to be subscribed to the list. I thus suggest to remove it.

Reply:
taken from ./CMakeLists.txt:set( NEST_USER_EMAIL "nest_user@nest-initiative.org" )
So you should remove it from your sources, or update it?

it's also listed in:
./doc/sli_docs/an-introduction-to-sli.rst:     Mailing list: nest_user@nest-initiative.org
./doc/guides/using_nest_with_music.rst:      Mailing list: nest_user@nest-initiative.org
./doc/guides/using_nest_with_music.rst:      Mailing list: nest_user@nest-initiative.org

Reply:
Thanks for bringing this up. The changes have been included in #1346 .

Reply:
Sending to the mailing list address will yield a mail containing subscription information as well as a contact path to the mailing list owners, so this is not so bad after all.